### PR TITLE
7149 - Icon adjustments in compact mode

### LINKED ISSUE #7149: Form: Compact Mode Issues
**Describe the bug**
**Issue 1:** Spinbox not rendered properly
**Issue 2:** Icon folder and close button look off in compact mode, mobile view.

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/form/example-compact-mode.html
2. Scroll down to Spinbox and Short Size
3. Click Icon folder
4. Add file
5. See error

**Expected behavior**
Spinbox, Icon folder and close button should be rendered properly.

**Version**

- ids-enterprise: 4.71.0-dev

**Screenshots**
**Issue 1:**
image

**Issue 2:**
image

**Platform**
 - BrowserStack
 - iPhone 14

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Spinbox: I can't find the border issue in local. The issue might have been fixed or I'm missing something
Fileupload: The icon alignment looked okay though I changed the size of the X icon since it looked bigger than the file icon

**Related github/jira issue (required)**:

Closes #7149 

**Steps necessary to review your pull request (required)**:

- Pull this branch, build and run the app
- Go to: http://localhost:4000/components/form/example-compact-mode.html
- Check spinbox
- Upload a file with a long name and check icon alignment

**Included in this Pull Request**:
- [ ] A note to the change log.

### CODE REVIEW

**Review by deep7102 — DISMISSED** [review]

**Review by tmcconechy — CHANGES_REQUESTED** [review]
I see some of the trigger icons are not centered vertically...like the date and time and lookup. Could maybe use an adjustment:

image

File: src/components/lookup/_lookup-new.scss [hunk 1]
```diff
@@ -54,7 +54,7 @@
 .form-layout-compact .field {
   .lookup-wrapper {
     .trigger {
-      margin-top: 4px;
+      margin-top: 6px;
```
Comment (andriivarhanov) [inline src/components/lookup/_lookup-new.scss:57]: it seems like the icons are closer to the bottom of the field now, maybe let's try 5px, here and for the similar icons

Comment (andriivarhanov) [inline src/components/lookup/_lookup-new.scss:57]: it seems like the icons are closer to the bottom of the field now, maybe let's try 5px, here and for the similar icons

**Comment by tmcconechy** [comment]
@yohannahbautista while we are getting this right some small things i would be willing to skip if needed

1) the hover state isnt centered (could be a little bigger if needed)
image
2) the file upload one has no hover state

**Review by tmcconechy — APPROVED** [review]

**Review by andriivarhanov — APPROVED** [review]

**Pull request merged**